revdb/revdb.gyp

```diff
@@ -488,6 +488,10 @@
 							# Error in ../../thirdparty/libsqlite/include/qry_dat.h
 							'-Werror=return-type',
 						],
+						'libraries':
+						[
+							'-lm',
+						],
 					},
```

Comment:
@livecodeali @runrevmark Do we need to do something similar to revzip/revzip.gyp? @montegoulding has reported a similar failure for revZip in the bug report.